validator/src/journal/publisher.rs

```diff
@@ -1,12 +1,124 @@
-use std::collections::VecDeque;
+use proto::batch::Batch;
+use std::collections::{HashSet, VecDeque};
+use std::mem;
+use std::slice::Iter;
+
+/// Ordered batches wating to be processed
```

Comment:
Hi @RyanLassigBanks , I accidentally put this on #1660 which is built off of this one. `wating` is missing an `i`: `waiting`.

Reply:
Fixed in 1660